janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticsearchRunner.java

```diff
@@ -32,7 +32,7 @@
  */
 public class ElasticsearchRunner extends DaemonRunner<ElasticsearchStatus> {
 
-    private static final String DEFAULT_HOME_DIR = ".";
+    private static final String DEFAULT_HOME_DIR = System.getProperty("user.dir");
```

Comment:
@amcp This isn't for the index directory but the home directory of the Elasticsearch distribution used for tests. This will be the build (e.g. target) directory relative to the project (janusgraph-es, janusgraph-dist-hadoop-2, etc.) home.